src/main.rs

```diff
@@ -1,5 +1,9 @@
 use std::io;
 
+fn count_overlap(secret: &str, guess: &str) -> usize {
+    secret.chars().filter(|letter| guess.contains(*letter)).count()
```

Comment:
(note: I have no idea why this `*` is necessary but it doesn't work without it 🤷‍♂️